DependencyInjection/DoctrineExtension.php

```diff
@@ -396,6 +396,14 @@ protected function ormLoad(array $config, ContainerBuilder $container)
                 $def->addTag('doctrine.event_subscriber');
             }
         }
+
+        if (!class_exists('Symfony\Bridge\Doctrine\PropertyInfo\DoctrineExtractor')) {
```

Comment:
If the component is not installed, those services will be removed as they are private and unused.